### PR TITLE
Makes radiation storms more blowout-esque

### PR DESCRIPTION
**Pull request by calibraptor**
Complete with screen shaking and fun sounds to scare the crap out of people experiencing it, also  the radiation is strong enough to significantly fuck you up if you don't get your ass into maintenance.

Only element I couldn't include was the entire station changing color as the storm blows through. Hoping someone more experienced can add that later on.

https://www.youtube.com/watch?v=UdXX2JDRhU0&feature=youtu.be

### CODE REVIEW

**Comment by GenericGoose** [comment]
da

File: html/changelogs/Calibraptor - Blowouts.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (Carbonhell) [inline html/changelogs/Calibraptor - Blowouts.yml]: You forgot to edit the name

**Comment by ghost** [comment]
Spooky.
This can't 1 hit kill someone can it?

**Comment by calibraptor** [comment]
@McBawbaggings Nope. Tested it myself. Granted you'll be in pretty bad shape if you don't seek out treatment for the resulting toxin damage.

**Comment by ghost** [comment]
Coolio.
I like it. +1

**Comment by Kierany9** [comment]
Maybe reduce the shaking just a little? That looks more like an earthquake than anything else.

**Comment by calibraptor** [comment]
Still has the same amount of shaking it had on old hippie. The point of it is to be pants shittingly scary.

**Comment by Badmex** [comment]
Will we see more stuff like this in the future becaues its cool. Also +1/3

**Comment by dreamcrab** [comment]
this is way better than the one we have now +6666666666666

**Comment by Spacedong** [comment]
can it have a small percent chance to forego the usual message and replace it with "BLOWOUT SOON, FELLOW STALKER!" ???

**Comment by TDcoolguy300** [comment]
Before i code review this, i'm going to get that fucking overlay working

**Comment by TDcoolguy300** [comment]
Code's A+ now

**Comment by TDcoolguy300** [comment]
Codechecked and 2 heads approved, Merging.

**Pull request merged**